catalog/nsili/catalog-nsili-endpoint/src/test/java/org/codice/alliance/nsili/endpoint/NsiliEndpointTest.java

```diff
@@ -133,20 +153,117 @@ public void testStandingQueryMgr() throws Exception {
     @Test
     public void testMaxNumResults() throws Exception {
         int currMaxNum = nsiliEndpoint.getMaxNumResults();
+
         nsiliEndpoint.setMaxNumResults(100);
         int setNumResults = nsiliEndpoint.getMaxNumResults();
         assertThat(setNumResults, is(100));
+
         nsiliEndpoint.setMaxNumResults(currMaxNum);
     }
 
     @Test
-    public void testQuerySources() throws Exception {
-        String testSource = "source1";
-        List<String> existingSources = nsiliEndpoint.getQuerySources();
-        List<String> querySources = Arrays.asList(new String[] {testSource});
-        nsiliEndpoint.setQuerySources(querySources);
-        List<String> setSources = nsiliEndpoint.getQuerySources();
-        assertThat(setSources, hasItem(testSource));
+    public void testValidSetQuerySources() throws Exception {
+        Set<String> existingSources = new HashSet<>(nsiliEndpoint.getQuerySources());
+
+        Set<String> validSources = new HashSet<>(Arrays.asList(VALID_SOURCE_IDS));
+        nsiliEndpoint.setQuerySources(validSources);
+        assertThat(nsiliEndpoint.getQuerySources(), is(validSources));
+
+        nsiliEndpoint.setQuerySources(existingSources);
```

Comment:
I know this was already commented on, but we are creating and destroying `nsiliEndpoint` in the `@before` and `@after` methods. Are these lines doing anything useful?